src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction_Actions.cs

```diff
@@ -31,6 +31,7 @@ public override Task<Compilation> TransformCompilationAsync(Compilation oldCompi
                 }
 
                 public DocumentId DocumentId => _newState.Attributes.Id;
+                public override bool CanApplyToCompilationWithGeneratedDocuments => true;
```

Comment:
i think it would be good to doc the true cases.  It's not honestly obvious to me why TouchDocument would be applicable here.

Reply:
All cases are now documented.